kolibri/deployment/default/urls.py

```diff
@@ -24,7 +24,7 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('kolibri.core.urls')),
+    url(r'^kolibri/', include('kolibri.core.urls')),
```

Comment:
I like namespacing, but in order to simplify routing on the front-end, I'd like the *first* part of the url path to always indicate which js app you're in. Something like `/mycoolapp/foo/bar` and `/anotherapp/bar/baz`. Not that it couldn't work with e.g. `/kolibri/mycoolapp/foo/bar`, just want to let you know what the convention is AFAIK.